src/main/java/frc/trigon/robot/subsystems/pitcher/PitcherCommands.java

```diff
@@ -16,6 +16,22 @@ public static Command getDebuggingCommand() {
         );
     }
 
+    public static Command getPitchToAmpCommand() {
+        return new StartEndCommand(
+                RobotContainer.PITCHER::pitchToAmp,
+                RobotContainer.PITCHER::stop,
+                RobotContainer.PITCHER
+        );
+    }
+
+    public static Command getSetPitchForCloseShot() {
+        return new StartEndCommand(
+                RobotContainer.PITCHER::setPitchForCloseShot,
+                RobotContainer.PITCHER::stop,
+                RobotContainer.PITCHER
+        );
+    }
```

Comment:
Honestly, instead of having these two, I would just call `getSetTargetPitchCommand` with a constant pitch.